lib/modules/platform/gitea/index.md

```diff
@@ -15,13 +15,24 @@ Then let Renovate use your PAT by doing _one_ of the following:
 
 You must set `platform=gitea` in your Renovate config file.
 
+### Version 1.19.x and older
+
 The PAT should have these permissions:
 
 - `repo`
 - `read:user` and `read:email`
 
 If you use Gitea packages, add the `read:packages` scope.
 
+### Version 1.20.0 and newer
```

Comment:
Hi @morremeyer, thanks for your contributions!

I think you're duplicating work from this PR though:

- #23615

Please review your PR, and the linked PR, and see if we need to make changes to the table in #23615. 😉

Reply:
Oh wow, I didn't find that one earlier. Thanks for the hint.